src/modules/regionHandling/handler.ts

```diff
@@ -0,0 +1,55 @@
+import { MessageCallback } from '../../lib';
+import { EventHandler } from '../eventHandling/handler';
+import { IAddRegionListenerOptions, RegionScope } from './interface';
+
+export class RegionHandler extends EventHandler {
+
+  protected createHandler(
+    callback: MessageCallback,
+    properties: string[],
+    options: IAddRegionListenerOptions,
+    ): EventListener {
+    return (event) => {
+      if (options.stopPropagation) {
+        event.preventDefault();
+      }
+
+      if (options.stopPropagation) {
+        event.stopPropagation();
+      }
+      if (options.stopImmediatePropagation) {
+        event.stopImmediatePropagation();
+      }
+      const region = options.region;
+      const eventType = options.eventType;
+      if (!region) {
+        console.error('"region" was not passed into RegionHandler');
+        return;
+      }
+
+      let x = (<MouseEvent> event).clientX;
```

Comment:
Just a couple of thoughts:
1) shouldn't there be a check for `options.eventType` to ensure `event` is actually a `MouseEvent`? (the `eventType` is specified by the client of the API, when invoking `this.regiGlue.addEventListener()`)
2) what about unifying Pointer Events? (touch handling)